implementations/rust/ockam/ockam_command/src/kafka/outlet/create.rs

```diff
@@ -34,16 +35,16 @@ pub struct CreateCommand {
     /// Examples are `/service/my-outlet` or `my-outlet`.
     /// If not provided, `/service/kafka_outlet` will be used.
     /// You will need this address when creating a Kafka Inlet using `ockam kafka-inlet create`.
-    #[arg(default_value_t = kafka_default_outlet_addr(), value_parser = extract_address_value)]
-    pub name: String,
+    #[arg(default_value = DefaultAddress::KAFKA_OUTLET, value_parser = extract_address)]
```

Comment:
Used `default_value = DefaultAddress::KAFKA_OUTLET` instead of the Address type like `Address::from(DefaultAddress::KAFKA_OUTLET)` because the latter outputs `0#kafka_outlet` as default value of `name` in the message of the `kafka-outlet create --help` command, which seems undesirable.